tools/build_defs/native_tools/BUILD

```diff
@@ -18,7 +18,7 @@ make_tool(
 
 native_tool_toolchain(
     name = "built_make",
-    path = "make/bin/make",
+    path = "$(execpath :make_tool)/bin/make",
```

Comment:
seems a bit odd. I'd expect `$(execpath :make_tool)` to be a path to the tool and `target = ":make_tool"` should be sufficient. Do you think it'd be difficult to update this to not require both `target` and `path`?

Reply:
`$(execpath :make_tool)` is the output from a `rules_foreign_cc` build which doesn't generate an executable target directly ( you have to do the dance of creating a `filegroup` and an executable target via a wrapper or another `cc_*` rule. This issue is along the lines of the same problem that #428 is looking to address too; there may be multiple tools required in the `PATH` but I guess in the emscripten case that this could be done by a shell wrapper that sets up the path so that `binaryen` is present in the context that `emscripten` is expecting it to be.

Also note that I'd made this observation in the `tool_access.bzl` file that it'd be easier if this was an executable target than a freeform path as we need to special case between absolute and relative paths.